src/main/java/eventManagement/MockCustomerService.java

```diff
@@ -0,0 +1,48 @@
+package eventManagement;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class MockCustomerService implements CustomerService {
+
+	
+	//mock data for customers
+	static ArrayList<Customer> customers = new ArrayList<Customer>();
+	
+	static {
+		Customer e1 = new Customer("Bob", "bob@gmail.com", "Begi555nner");
+		Customer e2 = new Customer("Tom", "jesse@gmail.com", "123Intermediate");
+		Customer e3 = new Customer("JPotato", "potato@gmail.com", "4dvacned");
+		customers.add(e1);
+		customers.add(e2);
+		customers.add(e3);
+	}
+
+	@Override
+	public Collection<Customer> getCustomers() {
+		return customers;
+	}
+
+	@Override
+	public void update(int id, Customer Customer) {
```

Comment:
Do we not implement update method at all? We can do something like "customers.add(id,customer)"? See the link: https://github.com/lyn-Jia/EventManagementProject/blob/main/src/main/java/eventManagement/RestAPI.java#L45

Reply:
@lyn-Jia

Reply:
Yea, it is just mocked for now. We will implement it later when we have a proper DB.